src/io/io.asm

```diff
@@ -0,0 +1,10 @@
+GLOBAL outb ; make the label outb visible outside this file
+; outb - send a byte to an I/O port
+; stack: [esp + 8] the data byte
+; [esp + 4] the I/O port
+; [esp ] return address
+outb:
```

Comment:
can we rename this function? maybe `write_io_port_byte` or something descriptive?

Reply:
renamed to io_port_write